cmd/config-versions.go

```diff
@@ -781,3 +781,30 @@ type serverConfigV29 struct {
 	// Logger configuration
 	Logger loggerConfig `json:"logger"`
 }
+
+// serverConfigV29 is just like version '29', the KMS configuration now contains an additional field `auto-encryption`
```

Comment:
serverConfigV29 is just like version '29' <- should be 30

Reply:
done